database/fixtures.sql

```diff
@@ -0,0 +1,193 @@
+BEGIN;
+
+
+INSERT INTO "users" (username, email, password)
+VALUES
+  ('DanielEineving', 'daniel@eineving.se', 'danieleineving'),
+  ('JimmyBerlin', 'berlin.jimmy@gmail.com', 'jimmyberlin'),
+  ('MikaelLonn', 'mickelonn93@gmail.com', 'mikaellonn'),
+  ('ChristopherAkersten', 'christopher.akersten@gmail.com', 'christopherakersten'),
+  ('JohanAndersson', 'lol2kpe@gmail.com', 'johanandersson');
+
+COMMIT;
+
+INSERT INTO token (token, userid)
+VALUES ('testtoken', (SELECT id
+                      FROM "users"
+                      WHERE username = 'ChristopherAkersten'
+                      LIMIT 1));
+
+COMMIT;
+
+INSERT INTO customer (name, email) VALUES ('Testbolag', 'test@testbolag.se');
+INSERT INTO customer (name, email) VALUES ('Chalmers n Stuff', 'gangsters@tjalmers.se');
+
+INSERT INTO materialdata (name, supplierName, initialAmount) VALUES
+  ('Stenhårt malm', 'JimmyBs stenhårda gäng', 100),
+  ('Dagis Tenn', 'Johans leksaksoutlet', 80);
+
+
+INSERT INTO orderdata (customerID, date) VALUES
+  ((SELECT id
+    FROM customer
+    WHERE name = 'Testbolag'), '2017-10-10'),
+  ((SELECT id
+    FROM customer
+    WHERE name = 'Testbolag'), '2017-11-23'),
+  ((SELECT id
+    FROM customer
+    WHERE name = 'Chalmers n Stuff'), '2017-09-19');
+
+INSERT INTO digitalpart (customerID, name) VALUES
+  ((SELECT id
+    FROM customer
+    WHERE name = 'Testbolag'), 'Screw'),
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal 7 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=plsql%3AS1192)